### ISSUE TITLE
Node 14 is not supported

### ISSUE THREAD

**Issue by anuragphadnis** [kyndryl-open-source/hashi-vault-js#32]
**Describe the bug**
The documentation says that the minimum node version required is 14 but when I tried  to install this dependency in node14 I get error that node 14 is not supported minimum support is for node 16.

**To Reproduce**
1. use node 14 (nvm use 14)
2. Install hashi-vault-js (yarn add hashi-vault-js)
3. error hashi-vault-js@0.4.12: The engine "node" is incompatible with this module. Expected version ">=16". Got "14.20.0

**Expected behavior**
Successful installation for node 14

**Environment:**
* Node.js version:  14.20.0
* yarn version: 1.22.19
* hashi-vault-js version: 0.4.12
* Vault Server Version (retrieve with `vault status`): NA
* Vault CLI Version (retrieve with `vault version`): NA
* Client/Server Operating Systems: Ubuntu

**Additional context**
I think this can be fixed easily by changing package.json
replacing 
```
  "engines": {
    "node": ">=16"
  },
```
with 
```
  "engines": {
    "node": ">=14"
  },
```

Or if node 16 is the minimum requirement updating the same in the readme.md

**Comment by rod-anami** [comment]
Thanks @anuragphadnis ! Your fix is correct, let me give some thoughts on what's the best for the project.

**Comment by anuragphadnis** [comment]
sure.

**Comment by anuragphadnis** [comment]
Hi @rod4n4m1 Any updates on this? If you want I can open PR for making node 14 as minimum required version?

**Comment by rod-anami** [comment]
Sorry @anuragphadnis, I thought I had this one resolved. Let's honor the current README as it was published on npm platform. I didn't see any great problem with v14. Please open a PR to fix the package.json if possible. I will do that if you're not able to. I'm tagging this to `0.4.13`.

**Comment by anuragphadnis** [comment]
sure @rod4n4m1, I will create a PR.

Do I need to follow any steps before creating PR like running test cases and if yes can you guide me in doing that?

Also, I can see that the package-lock.json contains a field named "package" which is storing the contents of package.json inside package-lock.json. However, when I run npm install it does not generate any such field can you tell me what am I missing due to which this field is not being generated?

Node version: v14.20.0
Npm version: 6.14.17

**Comment by rod-anami** [comment]
Nope, this is probably from older versions. Go ahead and create a PR for package.json only. package-lock.json is auto-generated, you can force it with `npm i --package-lock-only`, but don't need to include in the PR.

**Comment by anuragphadnis** [comment]
@rod4n4m1 

I have added the [PR](https://github.com/rod4n4m1/hashi-vault-js/pull/34)

**Comment by rod-anami** [comment]
Fixed on `0.4.13`
